app/imports/ui/components/ProfileSkillsDropdown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Dropdown, Header, Loader, Container } from 'semantic-ui-react'
+import PropTypes from 'prop-types';
+import { withTracker } from 'meteor/react-meteor-data';
+import { Meteor } from "meteor/meteor";
+import { Skills } from '../../api/skill/skill';
+
+class ProfileSkillsDropdown extends React.Component {
+  render() {
+    return (this.props.ready) ? this.renderDropdown() : <Loader active>Retrieving data</Loader>;
+  }
+
+
+  skillOptions = this.props.skills.map((skill) => ({
+    key: skill._id,
+    text: skill.name,
+    value: skill.name,
+    description: skill.description,
+  }));
+
+
+  // render() {
```

Comment:
remove this commented out render since we wont be using it

Reply:
done